plink-front/src/app/menu/menu.component.html

```diff
@@ -0,0 +1,4 @@
+<mat-tab-group mat-align-tabs="center" class="menu-container">
```

Comment:
I think the expectation regarding the outcome of this exercise is to create a custom styled component

Reply:
nevermind this comment.